.ci/jenkins/api-py3

```diff
@@ -1 +0,0 @@
-api
```

Comment:
I'd suggest a rebase to get rid of the changes in the `.ci/` directory.

Reply:
+1 - if you need help with the rebase process @assuntad23 let me know.